_SQLServer/TableLoadAbilities.sql

```diff
@@ -1,6 +1,193 @@
+
+
+--Hearthsworn Fighter
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Dwarven Curse', 'Action', 'Support', 2, -1, -1, 'Wave 3, Pull 3, Attack vs WILL', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Cleave', 'Action', 'Offense', 1, -1, -1, 'Sweep 1, Dangerous', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Hero''s Balm', 'Potion', 'Support', -1, -1, -1, 'The current attack is applied to the Hero affected by Hero''s Balm as though they were the target of the attack instead.', '1.0');--? Check values
+
+
+--Hexcast Sorceress
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Frozen Feet', 'Action', 'Offense', 1, -1, -1, '+1B WILL, Cold', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Hobbled Hands', 'Action', 'Offense', 1, -1, -1, '+1B WILL< Weak', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Broken Bones', 'Action', 'Offense', 1, -1, -1, '+1B WILL, Fragile', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Wizard Wings', 'Potion', 'Support', -1, -1, -1, '+1B WILL, Fly', '1.0');--? Check values
+
+
+--Miserable Toad
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Sticky Tongue', 'Action', 'Offense', 1, -1, -1, 'Missile 2, vs DEX.  If successful, Miserable Toad gains either one heart or one potion, in addition to any that are rolled.', '1.0');--? Check values
+
+
+--Nyan-Nyan
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Teeth & Fur', 'Action', 'Offense', 2, -1, -1, '+1R Attack, Knockdown, Pounce', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Purr', 'Action', 'Support', 1, -1, -1, 'Wave 1, Compel 2', '1.0');--? Check values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, KeywordVersion) VALUES ('Catnip Bomb', 'Potion', 'Support', 1, -1, -1, 'B Range 6, Burst 2, Pacify', '1.0');--? Check values
```

Comment:
AbilityDescription should read 'Bac Range 6'.
AbilityRange should be 6.
AbilityAttribute is technically 0